apps/sim-core/packages/core/package.json

```diff
@@ -0,0 +1,160 @@
+    "start": "yarn codegen && yarn prod-env webpack-dev-server --mode production",
+    "codegen": "graphql-codegen --config codegen.yml # core",
+    "fmt": "prettier \"*.{ts,tsx,js,jsx,json,css,scss}\" \"{scripts,src}/**/*.{ts,tsx,js,jsx,json,css,scss}\" --write; eslint --quiet --fix \"*.{ts,tsx,js}\" \"{scripts,src}/**/*.{ts,tsx,js}\"",
+    "fmt-check": "prettier \"*.{ts,tsx,js,jsx,json,css,scss}\" \"{scripts,src}/**/*.{ts,tsx,js,jsx,json,css,scss}\" --check || exit 1; eslint --quiet \"*.{ts,tsx,js}\" \"{scripts,src}/**/*.{ts,tsx,js}\"",
+    "deploy": "ts-node --project scripts/tsconfig.json scripts/deploy.ts",
+    "g": "ts-node --project scripts/tsconfig.json scripts/cli",
+    "test": "jest --forceExit --testPathIgnorePatterns '/node_modules/|/tests/e2e/'"
+  },
+  "jest": {
+    "preset": "ts-jest/presets/js-with-babel",
+    "globals": {
+      "ts-jest": {
+        "babelConfig": true
+      }
+    },
+    "moduleNameMapper": {
+      "^.+\\.(css|less|scss|sass)$": "identity-obj-proxy",
+      "^monaco-editor$": "<rootDir>/../../node_modules/monaco-editor/esm/vs/editor/editor.main.js",
+      "@hashintel/engine-web": "<rootDir>/src/util/simulation/mock-coreweb.ts"
+    },
+    "setupFilesAfterEnv": [
+      "<rootDir>/src/setupTests.ts"
+    ],
+    "transformIgnorePatterns": [
+      "/node_modules/(?!monaco-editor).+\\.js$"
+    ]
+  },
+  "// browserslist": [
+    "this list is an `intersection` of browsers (organized as they are in `https://caniuse.com/`",
+    "for easier reference) that support the features we need/use, namely (really wish this was a",
+    "`featureslist` instead):",
+    "",
+    "We have compiled a browserlist query based on the below at https://docs.google.com/spreadsheets/d/1pCjtbvxIgnbzIbgfrJuYIk6TLHhLhqIxNAA2Iord1gE/edit#gid=0",
+    ""
+  ],
+  "browserslist": [
+    "edge >= 79",
+    "firefox >= 67",
+    "chrome >= 66",
+    "safari >= 12.1"
+  ],
+  "// dependencies": [
+    "We compile to a single-page app, so technically everything should be in '/devDependencies'",
+    "But for the sake of organization, we put our runtime dependencies here."
+  ],
+  "dependencies": {
+    "@deck.gl/core": "8.3.7",
+    "@deck.gl/layers": "8.3.7",
+    "@fluentui/react": "7.150.0",
+    "@fullstory/browser": "1.4.5",
+    "@hashintel/engine-web": "0.1.1",
+    "@hashintel/utils": "0.0.1",
+    "@juggle/resize-observer": "3.2.0",
+    "@loaders.gl/core": "*",
+    "@luma.gl/core": "8.3.1",
+    "@material-ui/core": "4.11.4",
+    "@material-ui/lab": "4.0.0-alpha.56",
+    "@msrvida/sanddance-explorer": "3.1.0",
+    "@reduxjs/toolkit": "1.5.0",
+    "@sentry/browser": "6.2.0",
+    "@sentry/fullstory": "1.1.5",
+    "@sentry/integrations": "6.2.0",
+    "@svgr/core": "5.5.0",
+    "@svgr/plugin-jsx": "5.5.0",
+    "@svgr/plugin-svgo": "5.5.0",
+    "@testing-library/user-event": "12.6.2",
+    "@types/js-levenshtein": "1.1.0",
+    "@types/line-column": "1.0.0",
```

Comment:
Oughtn't these be devDeps?

Reply:
I'd expect so...  looks like they were added over time years ago. 

The comment above is helpful:
`"// dependencies": [
    "We compile to a single-page app, so technically everything should be in '/devDependencies'",
    "But for the sake of organization, we put our runtime dependencies here."
  ],`
  
  ...so there's no effect, it's just an organizational choice.  Moving them....